src/Extension/Traits/FluentObjectTrait.php

```diff
@@ -109,12 +115,48 @@ protected function updateFluentCMSFields(FieldList $fields)
             'Locale' => 'Locale'
         ];
 
+        // Augment Localisation tab with clickable locale links to allow easy navigation between model localisations
+        if ($owner->hasExtension(CMSEditLinkExtension::class)) {
+            $columns->setFieldCasting([
+                // Make sure our links are correctly rendered (valid HTML)
+                'Title' => 'HTMLVarchar->RAW',
+            ]);
+
+            $controller = Controller::has_curr() ? Controller::curr() : null;
+            $request = $controller?->getRequest();
+
+            // Pass getVars separately so we can process them later
+            $params = $request?->getVars() ?? [];
+
+            // This is to get URL only, getVars are not part of the URL
+            $url = $owner->CMSEditLink();
+            $url = Director::makeRelative($url);
+
+            $summaryColumns['Title'] = [
+                'title' => 'Title',
+                'callback' => function (Locale $object) use ($url, $params) {
+                    if (!$object->RecordLocale()) {
+                        return null;
+                    }
+
+                    $recordLocale = $object->RecordLocale();
+                    $locale = $recordLocale->getLocale();
+                    $params['l'] = $locale;
+                    $localeLink = Controller::join_links($url, '?' . http_build_query($params));
+                    $localeTitle = Convert::raw2xml($recordLocale->getTitle());
+                    $render = sprintf('<a href="%s" target="_top">%s</a>', $localeLink, $localeTitle);
+
+                    return $render;
```

Comment:
I tried this with `return DBField::create_field('HTMLVarchar', $render);` and that works for me locally.
When I checked out the `7.0` branch I found that that works for me as well.

Did you replicate the original issue in a fresh installation?

Reply:
yup, tested this with vanilla install with this composer file (ran composer update just before the test):

```
{
    "name": "silverstripe/installer",
    "type": "silverstripe-recipe",
    "description": "The SilverStripe Framework Installer",
    "require": {
        "php": "^8",
        "silverstripe/recipe-plugin": "^2",
        "silverstripe/recipe-cms": "~5.x-dev",
        "silverstripe-themes/simple": "~3.2.0",
        "silverstripe/login-forms": "^5",
        "dnadesign/silverstripe-elemental": "^5",
        "maxime-rainville/anyfield": "^0.0.0",
        "silverstripe/tagfield": "^3.1",
        "silverstripe-terraformers/gridfield-rich-filter-header": "dev-master",
        "tractorcow/silverstripe-fluent": "^7"
    },
    "require-dev": {
        "silverstripe/recipe-testing": "^3",
        "silverstripe/graphql-devtools": "^1.0"
    },
    "extra": {
        "resources-dir": "_resources",
        "project-files-installed": [
            "app/.htaccess",
            "app/_config.php",
            "app/_config/mimevalidator.yml",
            "app/_config/mysite.yml",
            "app/src/Page.php",
            "app/src/PageController.php",
            "behat.yml",
            "phpcs.xml.dist",
            "phpunit.xml.dist"
        ],
        "public-files-installed": [
            ".htaccess",
            "index.php",
            "web.config"
        ]
    },
    "repositories": {
        "ichaber/silverstripe-swiftype": {
            "type": "git",
            "url": "https://github.com/silverstripe-terraformers/silverstripe-swiftype.git"
        }
    },
    "config": {
        "process-timeout": 600,
        "allow-plugins": {
            "composer/installers": true,
            "silverstripe/vendor-plugin": true,
            "silverstripe/recipe-plugin": true
        }
    },
    "prefer-stable": true,
    "minimum-stability": "dev"
}

```

Reply:
It's very important to note that that is **not** a vanilla installation - there are a lot of optional modules there which you won't get when you just do `composer create-project silverstripe/installer && composer require tractorcow/silverstripe-fluent`

With this set of modules I do experience the problem, but not when I just do a truly vanilla installation. I suspect one of the additional optional modules in this set is interfering with the way the value is displayed.

Reply:
Aha! It's the constraint for `silverstripe/recipe-cms` here that changes things.

`"silverstripe/recipe-cms": "5.1.x-dev"` works as expected without this PR
`"silverstripe/recipe-cms": "~5.x-dev"` does not work as expected without this PR

Looks like there's a regression in framework or somewhere similar, and the problem should be addressed wherever that root cause regression lies.

Reply:
Excellent find @GuySartorelli ❤️ . I will change this PR into enhancement only covering the part that expands the display of title links which was my original intention. I'll check if I can find the source of the regression separately.